jcl/source/windows/JclDebug.pas

```diff
@@ -307,14 +308,16 @@   TJclBinDebugGenerator = class(TJclMapScanner)
   end;
 
   TJclBinDbgNameCache = record
-    Addr: DWORD;
-    FirstWord: Integer;
-    SecondWord: Integer;
+    Addr       : DWORD;
+    FirstWord  : Integer;
+    SecondWord : Integer;
+    Text       : String;
```

Comment:
`string` should be all lower cased.
Field types should not be indented, leave the first three alone and declare the new one like this:
`Text: string;`